### DISCUSSION TITLE
ERR_SSL_VERSION_OR_CIPHER_MISMATCH

### DISCUSSION THREAD

**Discussion by pedroresende** [vercel/vercel#6018] [Help]
After migrating my cname to vercel instead of correctly opening the page it's displaying ERR_SSL_VERSION_OR_CIPHER_MISMATCH . Any ideias ? Thanks in advance

**Comment by surjithctly** [comment]
Same issue here. Did you resolve it?

**Reply by pedroresende** [reply]
@surjithctly you have to change the proxy status from Proxied to DNS only. Please let me know if it worked

**Reply by surjithctly** [reply]
Can you point to me where should I change it?

**Reply by pedroresende** [reply]
You just have to click on it 
![Screenshot 2021-04-23 at 08 15 24](https://user-images.githubusercontent.com/951377/115833592-21e2b300-a40c-11eb-9b0e-07211afac2ea.png)

**Reply by surjithctly** [reply]
I'm not using Cloudflare :)

**Reply by surjithctly** [reply]
Found it is happening with NextDNS. When I disable that, it works. So nothing to do with vercel I guess. 

Thanks again :)
